docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+processor:
+  build: ../data-processor/.
```

Comment:
You don't need `.` on the ends of these. `.` is just an alias for "current directory" so when you specify the path to the directory, it's redundant.